src/m365/spo/commands/listitem/listitem-retentionlabel-remove.ts

```diff
@@ -0,0 +1,189 @@
+import { AxiosRequestConfig } from 'axios';
+import { Cli } from '../../../../cli/Cli';
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { formatting } from '../../../../utils/formatting';
+import { validation } from '../../../../utils/validation';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+export interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+  listUrl?: string;
+  listItemId: string;
+  confirm?: boolean;
+}
+
+class SpoListItemRetentionLabelRemoveCommand extends SpoCommand {
+  public get name(): string {
+    return commands.LISTITEM_RETENTIONLABEL_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Clear the retention label from a list item';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        listId: typeof args.options.listId !== 'undefined',
+        listTitle: typeof args.options.listTitle !== 'undefined',
+        listUrl: typeof args.options.listUrl !== 'undefined',
+        recycle: !!args.options.recycle,
+        confirm: !!args.options.confirm
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-u, --webUrl <webUrl>'
+      },
+      {
+        option: '-i, --listItemId <listItemId>'
+      },
+      {
+        option: '-l, --listId [listId]'
+      },
+      {
+        option: '-t, --listTitle [listTitle]'
+      },
+      {
+        option: '--listUrl [listUrl]'
+      },
+      {
+        option: '--recycle'
```

Comment:
Recycle is not a valid option in this command. Copy paste issue I think? :-)

Reply:
indeed, a copy past issue :smile: